aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -366,18 +384,24 @@ static void slow_conv2d_backward_weight_out_cpu_template(
 
   auto input = input_.contiguous();
```

Comment:
this looks unnecessary, you only need `input.size(0)` and `input.scalar_type()` below, both can be called on `input_`.

Reply:
Agreed, removed.